rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
+5. The Secretary or the President may give a copy of the representations to each member of the Association or, if they are not so given, the member may require that they be read out at the meeting at which the special resolution is to be proposed.
 
-## 22. Office holders
+#### 55 Vacation of office
 
-1. The officers of the Association shall be—
-  1. a President; 
-  2. a Vice-President; 
-  3. a Treasurer; and 
-  4. a Secretary. 
+1. A committee member may resign from the Committee by written notice addressed to the Committee.
 
-2. The provisions of section ‘Election of officers and ordinary committee members’, so far as they are applicable and with the necessary modifications, apply to and in relation to the election of persons to any of the offices referred to in subrule (1). 
+2. A person ceases to be a committee member if he or she—
+  1. ceases to be a member of the Association; or
+  2. fails to attend 3 consecutive committee meetings (other than special or urgent committee meetings) without leave of absence under rule 67; or
+  3. otherwise ceases to be a committee member by operation of section 78 of the Act.
 
-3. Each officer of the Association shall hold office until the annual or special general meeting roughly 12 months after the date of his or her election but is eligible for reelection. 
+  ##### Note
+  A Committee member may not hold the office of secretary if they do not reside in Australia.
 
-4. In the event of a casual vacancy in any office referred to in subrule (1), the committee may appoint one of its members to the vacant office and the member appointed may continue in office up to and including the conclusion of the annual general meeting next following the date of the appointment. 
+#### 56 Filling casual vacancies
 
-## 23. Ordinary members of the committee
+1. The Committee may appoint an eligible member of the Association to fill a position on the Committee that—
+  1. has become vacant under rule 55; or
+  2. was not filled by election at the last annual general meeting.
 
-1. Subject to these Rules, each ordinary member of the committee shall hold office until the annual or special general meeting roughly 12 months after the date of election but is eligible for re-election. 
+2. If the position of Secretary becomes vacant, the Committee must appoint a member to the position within 14 days after the vacancy arises.
 
-2. In the event of a casual vacancy occurring in the office of an ordinary member of the committee, the committee may appoint a member of the Association to fill the vacancy and the member appointed shall hold office, subject to these Rules, until the conclusion of the annual general meeting next following the date of the appointment. 
+3. Rule 56 also applies to any committee member appointed by the Committee to fill a vacant position under subrule 1 or 2.
```

Comment:
There is a circular reference to Rule 56 here, but the relevant rule appears to be Rule 55 (Vacation of office). Can someone confirm this is correct and fix this reference?